src/plugins/telemetry/server/config.ts

```diff
@@ -46,6 +47,31 @@ export const configSchema = schema.object({
   sendUsageFrom: schema.oneOf([schema.literal('server'), schema.literal('browser')], {
     defaultValue: 'server',
   }),
+  logging: schema.object({
+    appenders: schema.mapOf(schema.string(), config.logging.appenders, { defaultValue: new Map() }),
+    loggers: schema.arrayOf(config.logging.loggers, { defaultValue: [] }),
+  }),
```

Comment:
This allows users to overwrite the _silent-by-default_ telemetry logs behaviour in production.